l5kit/l5kit/simulation/dataset.py

```diff
@@ -27,6 +28,7 @@ class SimulationConfig(NamedTuple):
     distance_th_close: float
     start_frame_index: int = 0
     num_simulation_steps: Optional[int] = None
+    disable_tqdm: bool = True
```

Comment:
nit: I would change that to `show_progress` or `show_info` just to make it general for print/logging later, otherwise if we use that flag later for logging/other prints we would have to change the naming here.